EESSI-extend-2023.06-easybuild.eb

```diff
@@ -0,0 +1,96 @@
+easyblock = 'Binary'
+
+name = 'EESSI-extend'
+version = '2023.06'
+# May have different ways to extend EESSI in future (manually, other tools,...)
+versionsuffix = '-easybuild'
+
+homepage = 'https://eessi.github.io/docs/'
+
+description = """
+ The goal of the European Environment for Scientific Software Installations
+ (EESSI, pronounced as "easy") is to build a common stack of scientific
+ software installations for HPC systems and beyond, including laptops,
+ personal workstations and cloud infrastructure.
+
+ This module allows you to extend EESSI using the same configuration for
+ EasyBuild as EESSI itself uses. The default installation is the users
+ home directory, but this can be overridden for a site installation (by setting
+ EESSI_SITE_INSTALL) or for a direct installation under CVMFS (by setting
+ EESSI_CVMFS_INSTALL). 
+"""
+
+toolchain = SYSTEM
+
+source_urls = None
+sources = ['eb_hooks.py']
```

Comment:
Where is this supposed to come from?

Reply:
The current directory. It's a bit of a problem though, as every update of `eb_hooks.py` should require a rebuild of this easyconfig. Instead, we should probably set an envvar (with a default path where we ship the hooks via EESSI).

Reply:
They're here:

https://github.com/EESSI/software-layer/blob/2023.06-software.eessi.io/eb_hooks.py

Reply:
I meant in the /cvmfs/sofgtware.eessi.io tree

Reply:
I asssumed that's what @ocaisa was talking about when he said "where we ship the ..."

Reply:
Ah, sorry. I don't think we store that file in our CVMFS repo at the moment.

Reply:
In the most recent commits I've added an `easybuild` directory underneath the `init` space where I store the hooks